plugins/caches.js

```diff
@@ -332,6 +333,96 @@ exports.plugin = {
                     }
                 }
             }
+        }, {
+            method: 'DELETE',
+            path: '/caches/{scope}/{id}',
+            handler: async (request, h) => {
+                if (strategyConfig.plugin !== 's3') {
+                    return h.response();
+                }
+
+                let cachePath;
+                const apiUrl = config.get('ecosystem.api');
+                const payload = {
```

Comment:
can we name this something else, like `options`? `payload` kinda confuses me

Reply:
`options` is defined in global scope, renaming it `opts`